foundry/entity.d.ts

```diff
@@ -323,7 +330,7 @@ declare class Entity<D extends Entity.Data = Entity.Data> {
   /**
    * Entity- specific actions that should occur when the Entity is first created
    */
-  protected _onCreate(data: D, options: any, userId: string): void;
+  protected _onCreate(data: DeepPartial<D>, options: any, userId: string): void;
```

Comment:
I think the `onCreate` method is called after the data is received back from the server, so it should be filled with defaults? so no `DeepPartial` needed?